compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -212,6 +212,58 @@ class ConvertNCHWToNHWC final : public luci::CircleNodeMutableVisitor<bool>
     throw std::runtime_error(node->name() + " is an unsupported operator.");
   }
 
+  bool visit(luci::CircleInput *node)
+  {
+    const auto n = node->dim(0).value();
+    const auto c = node->dim(1).value();
+    const auto h = node->dim(2).value();
+    const auto w = node->dim(3).value();
+
+    node->dim(1).set(h);
+    node->dim(2).set(w);
+    node->dim(3).set(c);
```

Comment:
When considering unknown dimensions, following codes

```suggestion
    const auto n = node->dim(0);
    const auto c = node->dim(1);
    const auto h = node->dim(2);
    const auto w = node->dim(3);

    node->dim(1) = h;
    node->dim(2) = w;
    node->dim(3) = c;
```

 or adding assert would be better.

Reply:
I've updated the code. Thanks.